File: Sever/BorrowBooks/BorrowBooks/Models/BorrowModel.cs
using System.ComponentModel.DataAnnotations;

namespace BorrowBooks.Models
{
    public class BorrowModel
    {
        [Key]
        public int Id { get; set; }
        public int BookId { get; set; }
        public string BookName { get; set; }
        public string UserId { get; set; }
        public string FullName { get; set; }
        public bool IsTrue { get; set; }
        public DateTime BorrowDate { get; set; } = DateTime.Now; // Ngày mượn
        public DateTime? ReturnDate { get; set; } 

    }
}

File: Client/src/app/Shared/models/borrow.model.ts
// src/app/models/book.dto.ts
export interface BookDTO {
  id: number;
  title: string;
  availableCopies: number;
}

// src/app/models/user.dto.ts
export interface UserDTO {
  id: string;
  fullName: string;
}

// src/app/models/borrow.model.ts
export interface BorrowModel {
  id: number;
  bookId: number;
  bookName: string;
  userId: string;
  fullName: string;
  isTrue: boolean;
  borrowDate: Date;
  returnDate?: Date;
}
